service/src/main/java/org/cbioportal/genome_nexus/service/internal/VariantAnnotationServiceImpl.java

```diff
@@ -70,6 +74,8 @@
 
     private final CachedVariantAnnotationFetcher cachedExternalResourceFetcher;
     private final CachedVariantIdAnnotationFetcher cachedVariantIdAnnotationFetcher;
+    private final CachedVariantRegionAnnotationFetcher cachedVariantRegionAnnotationFetcher;
```

Comment:
No we have 3 different data fetchers in a single class, and we determine which one to use by looking at the format parameter. This might be another good reason to have separate service implementations for separate formats, since seems like for each format we are using a separate data source. If we have 10 additional formats in the future, we may end up having a long list of fetchers and lengthy switch/case, if/else blocks.